nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/assets/styles/_listing-table.scss

```diff
@@ -110,15 +120,15 @@
             }
 
             tr:hover {
-                background-color: $surface-highlight !important;
+                background-color: $highlight-surface;
             }
 
             .selected {
-                background-color: $selected-row-color !important;
+                background-color: $highlight-surface !important;
```

Comment:
Ok yea, James also just pointed this out. It is especially tricky since we have the `unset` values in the properties table as well. His suggestion was to remove the even styles for the `.listing-table`'s and just have the lighter or darker nifi surface palette be the selected style depending on light/dark mode.